superduperdb/vector_search/interface.py

```diff
@@ -103,13 +104,14 @@ def find_nearest_from_array(
         :param n: number of nearest vectors to return
         """
         if CFG.cluster.vector_search.uri is not None:
-            response = request_server(
-                service='vector_search',
-                data=h,
-                endpoint='query/search',
-                args={'vector_index': self.vector_index, 'n': n},
-            )
-            return response['ids'], response['scores']
+            if CFG.cluster.vector_search.type != 'pg_vector':
+                response = request_server(
+                    service='vector_search',
+                    data=h,
+                    endpoint='query/search',
+                    args={'vector_index': self.vector_index, 'n': n},
+                )
+                return response['ids'], response['scores']
```

Comment:
The same here

Reply:
@jieguangzhou It's throwing error that's why, I did this